lib/t/matches.ex

```diff
@@ -357,6 +490,8 @@ defmodule T.Matches do
     m = "Saving slots offer for match: #{offerer_id} offered date  #{mate_id}"
     Bot.async_post_message(m)
 
+    match_timeslot_new_event(match_id, "saving slot")
```

Comment:
Best if event names don't have whitespace.

Reply:
`new` in function names usually means building a data structure in memory, not persisting to DB. If it's persisting to DB it should be renamed and moved to the same transaction as other ops below.